src/tests/functional/plugin/cpu/single_layer_tests/grid_sample.cpp

```diff
@@ -0,0 +1,343 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include <common_test_utils/ov_tensor_utils.hpp>
+
+using namespace CPUTestUtils;
+using namespace ov::test;
+using ov::op::v9::GridSample;
+
+namespace CPULayerTestsDefinitions {
+
+typedef std::tuple<
+        std::vector<InputShape>,                 // Input shapes
+        GridSample::InterpolationMode,           // Interpolation mode
+        GridSample::PaddingMode,                 // Padding mode
+        bool,                                    // Align corners
+        ElementType,                             // Data precision
+        ElementType,                             // Grid precision
+        CPUSpecificParams,                       // CPU specific params
+        std::map<std::string, std::string>       // Additional config
+> GridSampleLayerTestCPUParams;
+
+class GridSampleLayerTestCPU : public testing::WithParamInterface<GridSampleLayerTestCPUParams>,
+                           virtual public SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<GridSampleLayerTestCPUParams> obj) {
+        std::vector<InputShape> inputShapes;
+        GridSample::InterpolationMode interpolateMode;
+        GridSample::PaddingMode paddingMode;
+        bool alignCorners;
+        ElementType dataPrecision, gridPrecision;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
+
+        std::tie(inputShapes, interpolateMode, paddingMode, alignCorners, dataPrecision, gridPrecision, cpuParams, additionalConfig) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=(";
+        for (size_t i = 0lu; i < inputShapes.size(); i++) {
+            result << CommonTestUtils::partialShape2str({inputShapes[i].first}) << (i < inputShapes.size() - 1lu ? "_" : "");
+        }
+        result << ")_TS=";
+        for (size_t i = 0lu; i < inputShapes.front().second.size(); i++) {
+            result << "{";
+            for (size_t j = 0lu; j < inputShapes.size(); j++) {
+                result << CommonTestUtils::vec2str(inputShapes[j].second[i]) << (j < inputShapes.size() - 1lu ? "_" : "");
+            }
+            result << "}_";
+        }
+
+        result << "interpMode=" << (interpolateMode == GridSample::InterpolationMode::BILINEAR ? "BILINEAR" :
```

Comment:
Could you please add these modes in [ngraph::helper](/home/sidorova/work/INT8/openvino/src/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/utils/ngraph_helpers.hpp)?